bijection-hbase/src/test/scala/com/twitter/bijection/hbase/HBaseInjectionsSpecifications.scala

```diff
@@ -0,0 +1,161 @@
+ */
+class HBaseInjectionsSpecifications extends WordSpec {
+  "HBaseInjections" should {
+    "respect ImmutableBytesWritable offset and length" in {
+      val long: Long = 1
+      val double: Double = 2
+      val int: Int = 3
+
+      val longIBW = long2BytesWritableInj(long)
+      val doubleIBW = double2BytesWritableInj(double)
+      val intIBW = int2BytesWritableInj(int)
+
+      val allBytes = longIBW.copyBytes ++ doubleIBW.copyBytes ++ intIBW.copyBytes
+
+      val sharedLongIBW = new ImmutableBytesWritable(allBytes, 0, 8)
+      val sharedDoubleIBW = new ImmutableBytesWritable(allBytes, 8, 8)
+      val sharedIntIBW = new ImmutableBytesWritable(allBytes, 16, 4)
+
+      assert(Success(long) == long2BytesWritableInj.invert(sharedLongIBW))
+      assert(Success(double) == double2BytesWritableInj.invert(sharedDoubleIBW))
+      assert(Success(int) == int2BytesWritableInj.invert(sharedIntIBW))
+    }
+
+    "round trip String -> Array[Byte]" in {
+      val expected = "Bonjour le monde"
+      val bytes = Injection[String, Array[Byte]](expected)
+      val result = Injection.invert[String, Array[Byte]](bytes)
+      assert(Success(expected) == result)
+    }
+
+    "round trip Long -> Array[Byte]" in {
+      val expected = 42L
+      val bytes = Injection[Long, Array[Byte]](expected)
+      val result = Injection.invert[Long, Array[Byte]](bytes)
+      assert(Success(expected) == result)
+    }
+
+    "round trip Int -> Array[Byte]" in {
+      val expected = 42
+      val bytes = Injection[Int, Array[Byte]](expected)
+      val result = Injection.invert[Int, Array[Byte]](bytes)
+      assert(Success(expected) == result)
+    }
+
+    "round trip Double -> Array[Byte]" in {
+      val expected = 42.0
+      val bytes = Injection[Double, Array[Byte]](expected)
+      val result = Injection.invert[Double, Array[Byte]](bytes)
+      assert(Success(expected) == result)
+    }
+
+    "round trip Float -> Array[Byte]" in {
+      val expected = 42.0F
+      val bytes = Injection[Float, Array[Byte]](expected)
+      val result = Injection.invert[Float, Array[Byte]](bytes)
+      assert(Success(expected) == result)
+    }
+
+    "round trip Short -> Array[Byte]" in {
+      val expected = 1.toShort
+      val bytes = Injection[Short, Array[Byte]](expected)
+      val result = Injection.invert[Short, Array[Byte]](bytes)
+      assert(Success(expected) == result)
+    }
+
+    "round trip BigDecimal -> Array[Byte]" in {
+      val expected = BigDecimal(1)
+      val bytes = Injection[BigDecimal, Array[Byte]](expected)
+      val result = Injection.invert[BigDecimal, Array[Byte]](bytes)
+      assert(Success(expected) == result)
+    }
+
+    "round trip Boolean -> Array[Byte]" in {
+      val expected = true
+      val bytes = Injection[Boolean, Array[Byte]](expected)
+      val result = Injection.invert[Boolean, Array[Byte]](bytes)
+      assert(Success(expected) == result)
+    }
+
+    "round trip String -> ImmutableBytesWritable" in {
```

Comment:
this is testing your code, but the stuff above is just retesting existing stuff in bijection, right?